docs/getting-started/integrate-auth/react-native.mdx

```diff
@@ -0,0 +1,161 @@
+---
```

Comment:
Have you tried running this guide on your machine?

Reply:
I just checked it out and unfortunately, something seems to be wrong. I mean I can get the app to "boot" in the web mode, but other than clicking the <img width="195" alt="image" src="https://user-images.githubusercontent.com/29116492/179219035-9e77e9c7-466f-43ab-a1d6-29819e47a84d.png"> button there isn't anything else I can do. No fields to provide user email/password are shown. I also found that:

- By default, the app uses a playground project address that I guess is very old. The address is: `https://demo.tenants.staging.oryapis.dev/api/kratos/public`. I think we should change this to our current playground project URL. (I replaced it with my project's SDK URL and that didn't help). 
- Running `npm run web` works, but running the app in the web env. isn't possible from the Expo dash since `"web" is not included in the project app.config.js or app.json "platforms" array`. Easy fix. 

I wasn't able to run the app on simulated Android (duh) or iOS (Xcode is needed and doesn't want to install for some reason). 

After that fiasco I went back to the original blog post and ran every installation command. Still doesn't work. 

@aeneasr do you think you can have a look at the app and debug? If not, I can ask someone else from the team on Monday :)

Reply:
Ok thank you for checking, I probably won't be able to look into this this week. Someone with a bit of expertiese in this should look into it though, it's a bit tricky to set up XCode etc.

Should we remove this doc from the guide and move it to another PR so that we can merge the restructure and add the native guide once it works?

Reply:
Yes, let's do this. I'll move this doc to a separate PR 👍🏻